packages/package-builder/tests/Reflection/PrivatesAccessorTest.php

```diff
@@ -20,5 +21,22 @@ public function test(): void
 
         $fetchedParentValue = $privatesAccessor->getPrivateProperty($someClassWithPrivateProperty, 'parentValue');
         $this->assertSame($someClassWithPrivateProperty->getParentValue(), $fetchedParentValue);
+
+        $fetchedValue = $privatesAccessor->getPrivatePropertyOfClass($someClassWithPrivateProperty, 'object', stdClass::class);
+        $this->assertSame($someClassWithPrivateProperty->getObject(), $fetchedValue);
+    }
+
+    public function testSetter(): void
+    {
+        $privatesAccessor = new PrivatesAccessor();
+        $someClassWithPrivateProperty = new SomeClassWithPrivateProperty();
+
+        $privatesAccessor->setPrivateProperty($someClassWithPrivateProperty, 'value', 25);
+        $this->assertSame(25, $someClassWithPrivateProperty->getValue());
+
+        $newObject = new stdClass();
+        $this->assertNotSame($newObject, $someClassWithPrivateProperty->getObject());
+        $privatesAccessor->setPrivatePropertyOfClass($someClassWithPrivateProperty, 'object', $newObject, stdClass::class);
+        $this->assertSame($newObject, $someClassWithPrivateProperty->getObject());
     }
```

Comment:
Could you separate it to 2 new methods? So we have original code tested separately and new one also